script/functions.sh

```diff
@@ -184,7 +184,7 @@ function check_documentation_coverage {
 
 function check_style_and_lint {
   echo "bin/rubocop lib"
-  bin/rubocop lib
+  eval "RUBYOPT=$RUBYOPT:' --enable rubygems' bin/rubocop lib"
```

Comment:
FWIW, this method of switching rubygems back to being enabled didn't work on Ruby 2.5, so I had to change it to use `unset` in a subshell.  See this:

https://github.com/rspec/rspec-dev/pull/195/commits/3f380ca5b9590ee89c7ade1358b21f9f68bbc25b

Reply:
oh right you said that above, but I missed it and just copied from what I had locally :/